snapshot/cli/cli.go

```diff
@@ -195,7 +197,11 @@ func (c *createGitBundleCommand) run(db snapshot.DB, _ *cobra.Command, _ []strin
 			return fmt.Errorf("Couldn't create temp dir: %v", err)
 		}
 		tempDir = td.Dir
-		bundleFilename = path.Join(tempDir, fmt.Sprintf("bs-%v-%v.bundle", c.basis, c.ref))
+
+		// Don't use commit sha in case of collision with other bundle,
+		// create simple sha derived from basis and ref
+		sha := sha1.Sum([]byte(revList))
```

Comment:
I see now, I was thinking about Stdout/Stderr snapshots which call write-tree which results in a unique sha every time. Obviously we can't do the same thing for repo diff snapshots since that would mess with our history. So yeah, this is a good workaround.